openreview/venue/process/assignment_pre_process.js

```diff
@@ -0,0 +1,37 @@
+async function process(client, edge, invitation) {
+  client.throwErrors = true
+
+  const { groups } = await client.getGroups({ id: edge.domain })
+  const domain = groups[0]
+  const venueId = domain.id
+  const submissionName = domain.content.submission_name?.value
+  const reviewName = invitation.content.review_name?.value
+  const reviewersAnonName = invitation.content.reviewers_anon_name?.value
+
+  if (!reviewName) {
+    return
+  }
+
+  if (!edge.ddate) {
+    return
+  }
+
+  const { notes } = await client.getNotes({ id: edge.head })
+  const submission = notes[0]
+  const submissionGroupId = `${venueId}/${submissionName}${submission.number}`
+
+  const { groups: anonGroups, count: groupCount } = await client.getGroups({ prefix: `${submissionGroupId}/${reviewersAnonName}`, signatory: edge.tail })
+
+  if (groupCount.length == 0) { 
```

Comment:
i think this should be anonGroups.length=== or groupCount===

Reply:
fixed